src/ViewModel/CategoryPath.php

```diff
@@ -40,6 +40,11 @@ public function getValue(): string
         return implode(',', $value);
     }
 
+    public function getValue(): self
```

Comment:
reminder:
Using this with Magento 2.3 leads to
https://github.com/magento/magento2/issues/11905